db/migrate/20160307161402_add_scheduled_reports.rb

```diff
@@ -0,0 +1,13 @@
+class AddScheduledReports < ActiveRecord::Migration
+  def change
+    create_table :scheduled_reports do |t|
+      t.string :name, null: false
+      t.string :frequency, null: false
```

Comment:
did you consider using AR enums for this? would give us all the boolean methods (`weekly?` etc) for free